var/spack/repos/builtin/packages/parallel-netcdf/package.py

```diff
@@ -7,14 +7,32 @@
 
 
 class ParallelNetcdf(AutotoolsPackage):
-    """Parallel netCDF (PnetCDF) is a library providing high-performance
-    parallel I/O while still maintaining file-format compatibility with
-    Unidata's NetCDF."""
+    """PnetCDF (Parallel netCDF) is a high-performance parallel I/O
+    library for accessing files in format compatibility with Unidata's
+    NetCDF, specifically the formats of CDF-1, 2, and 5. The CDF-5
+    file format, an extension of CDF-2, supports unsigned data types
+    and uses 64-bit integers to allow users to define large
+    dimensions, attributes, and variables (> 2B array elements).
+    In addition to the conventional netCDF read and write APIs,
+    PnetCDF also provides a set of nonblocking APIs which 
+    allow users to post multiple read and write requests first,
+    and later let PnetCDF aggregate the requests into a large
+    MPI-IO request to achieve better performance."""
```

Comment:
Not sure if this much detail is needed in the description.

Reply:
Shortened.